runner/test/cases/server/plugins/engine/pageControllers/getPageController.test.ts

```diff
@@ -77,4 +79,74 @@ suite("Engine Page Controllers getPageController", () => {
+    );
+
+    test("it correctly includes/filters state values", () => {
+      const formModel = new FormModel(formJson, {});
+
+      //Selected page appears after convergence and contains a conditional field
+      //This is the page we're theoretically browsing to
+      const testConditionsPage = formModel.pages.find(
+        (page) => page.path === "/testconditions"
+      );
+
+      const page = new PageController(formModel, testConditionsPage?.pageDef);
+
+      //The state below shows we said we had a UKPassport and entered details for an applicant
+      let completeState = {
+        progress: [
+          "/csds/uk-passport?visit=7O4_nT1TVI",
+          "/csds/how-many-people?visit=7O4_nT1TVI",
+          "/csds/applicant-one?visit=7O4_nT1TVI",
+          "/csds/applicant-one-address?visit=7O4_nT1TVI",
+          "/csds/contact-details?visit=7O4_nT1TVI",
+        ],
+        checkBeforeYouStart: {
+          ukPassport: true,
+        },
+        applicantDetails: {
+          numberOfApplicants: 1,
+          phoneNumber: "1234567890",
+          emailAddress: "developer@test.com",
```

Comment:
```suggestion
          emailAddress: "developer@example.com",
```
Always safest to use `example.com` in test data, even if we don't think we'll send mail to it. It's a reserved domain, so will never resolve to a real domain - https://www.iana.org/domains/reserved